guidelines/index.html

```diff
@@ -2236,17 +2236,17 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
          </section>
          
          
-         <section class="informative" id="privacy-summary"><div class="header-wrapper"><h3 id="x5-6-privacy-considerations"><bdi class="secno">5.6 </bdi>Privacy Considerations</h3><a class="self-link" href="#privacy-summary" aria-label="Permalink for Section 5.6"></a></div><p><em>この節は規定ではない。</em></p>
+         <section class="informative" id="privacy-summary"><div class="header-wrapper"><h3 id="x5-6-privacy-considerations"><bdi class="secno">5.6 </bdi>プライバシーの考慮事項</h3><a class="self-link" href="#privacy-summary" aria-label="Permalink for Section 5.6"></a></div><p><em>この節は規定ではない。</em></p>
             
             
             
-            <p>Success Criteria within this specification which the Working Group has identified possible implications for privacy, either by providing protections for end users or which are important for web site providers to take in to consideration when implementing features designed to protect user privacy, are listed below. This list reflects the current understanding of the Working Group but other Success Criteria may have privacy implications that the Working Group is not aware of at the time of publishing.</p>
+            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供することによってプライバシーに影響を与える可能性があるとワーキンググループが特定したもの、又は、利用者のプライバシーを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要であるものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないプライバシーに関する影響があるかもしれない。</p>
```

Comment:
あまりにも英文法の難易度が高すぎますが、「ワーキンググループが特定したもの」は全体にかかっているように思います。たぶんこうで、

```
Success Criteria within this specification
which the Working Group has identified possible implications for privacy,

  either 
    by providing protections for end users
  or
    which are important for web site providers to take in to consideration when implementing features designed to protect user privacy,

are listed below.
```

言っていることを噛み砕くとこういうことのはず。

```
達成基準のうちいくつかは、プライバシーに影響を与える可能性がある。その理由は以下のいずれかである:

- その達成基準が、エンドユーザに(プライバシーの)保護を提供する
- プライバシー機能を実装する際に、その達成基準について考慮することが重要となる

ワーキンググループがこの基準に合致するとみなした達成基準を以下にリスト化した。
```

訳文としてはこうのような気がします。

```suggestion
            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供するか、又は、利用者のプライバシーを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要であり、プライバシーに影響を与える可能性があるとワーキンググループが特定したものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないプライバシーに関する影響があるかもしれない。</p>
```

わかりにくいですが、原文もわかりにくいのでヨシ! という方向で……
最悪訳註でフォローですが、まあここは正確な意味がわからなくても、雰囲気でいいような気はします。

Reply:
> 「ワーキンググループが特定したもの」は全体にかかっている

これは同意です。もんどさんが「either A or Bが成立していない」と書かれているのが気になっていますが…。

ひとまず取り込まさせてください。
もんどさん、補足あればお願いします…！

Reply:
@momdo さんいかがでしょうか?

Reply:
https://github.com/waic/wcag22/pull/47#pullrequestreview-1874560407 にて解決しています。